### PR TITLE
[DNM] Fixes crusher

### LINKED ISSUE #731: Borgs cannot use their crushers
## Testmerges:

[#456: 'Storyteller port from horizon [DNM]' by nevimer at commit aa3b527d63](https://github.com/Bubberstation/Bubberstation/pull/456)
[#625: 'Contractor and Ninja Modsuits have a deployable macrobomb (Enable the DNA lock and hit Lock on the modsuit to arm)' by nevimer at commit 6c1dd1cc91](https://github.com/Bubberstation/Bubberstation/pull/625)
[#306: 'Ports GAS' by nevimer at commit 9b5b12ae48](https://github.com/Bubberstation/Bubberstation/pull/306)
[#700: '[Mostly Modular] Hypnosis Preference' by StrangeWeirdKitten at commit 42a3878da0](https://github.com/Bubberstation/Bubberstation/pull/700)
[#719: '[Modular] Removes multiple secure tech boards from the research tree (Makes them orderable)' by StrangeWeirdKitten at commit 01a07248b2](https://github.com/Bubberstation/Bubberstation/pull/719)

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

I just barely have an idea of what I'm doing cyborg code is such a snowflake mess

closes #731

## Why It's Good For The Game

Bug fixed

## Proof Of Testing

I did test it and seems like it works without unintended behaviour

## Changelog

:cl:
fix: Cyborg crushers now function as intended
/:cl:

### CODE REVIEW

File: code/_onclick/cyborg.dm [hunk 1]
```diff
@@ -83,6 +83,14 @@
...
 			W.melee_attack_chain(src, A, params)
 			return
+		else
+			if(LAZYACCESS(modifiers, RIGHT_CLICK))
+				var/after_attack_secondary_result = W.afterattack_secondary(A, src, FALSE, params)
+
+				if(after_attack_secondary_result == SECONDARY_ATTACK_CALL_NORMAL)
+					W.afterattack(A, src, FALSE, params)
+			else
+				W.afterattack(A, src, FALSE, params)
```
Comment (StrangeWeirdKitten) [inline code/_onclick/cyborg.dm:93]:
```suggestion
// BUBBER EDIT - START
		else
			if(LAZYACCESS(modifiers, RIGHT_CLICK))
				var/after_attack_secondary_result = W.afterattack_secondary(A, src, FALSE, params)

				if(after_attack_secondary_result == SECONDARY_ATTACK_CALL_NORMAL)
					W.afterattack(A, src, FALSE, params)
			else
				W.afterattack(A, src, FALSE, params)
// BUBBER EDIT - END
```
May need this. I'm not entirely sure if the crusher itself is a skyrat or a TG thing. I'm leaning skyrat and I'm unsure if there's a need on TG itself to justify this being an upstream.

Comment (Majkl-J) [reply code/_onclick/cyborg.dm:93]: It's a TG issue actually and this is marked as DNM to see if it causes any issues before I upstream it

Comment (StrangeWeirdKitten) [reply code/_onclick/cyborg.dm:93]: > Reference

👍

Comment (pixelkitty286) [reply code/_onclick/cyborg.dm:93]: Cyborg crushers are a tg thing but the crusher itself is not

**Pull request closed without merging**